VCFv4.4.draft.tex

```diff
@@ -1701,6 +1706,40 @@ \subsection{Representing unspecified alleles and REF-only blocks (gVCF)}
 \end{flushleft}
 \normalsize
 
+
+\pagebreak
+\subsection{Representing copy number variation}
+\label{cnv}
+
+To encode copy number variation, VCF uses $<$CNV$>$, $<$DEL$>$ and $<$DUP$>$ symbolic structural variant alleles, CN INFO and FORMAT fields.
+
+Allele specific copy number is specified through a $<$CNV$>$ ALT allele for each distinct allelic copy number.
+POS and INFO SVLEN specify the genomic interval over which the copy number is defined.
+As with all symbolic structural variants, the starting position of the interval is the base immediately after POS.
+For example, a region on chr1 from position 101 to 300 (both inclusive) with allele-specific copy numbers of 1 and 2 can be represented as follows:
+
+\footnotesize
+\begin{verbatim}
+	chr1 100 . T <CNV>,<CNV> . . END=130;SVLEN=30,30;CN=1,2 GT:CN 1/2:3
+\end{verbatim}
+\normalsize
+
+All $<$CNV$>$ alleles in the same VCF record should have the same SVLEN.
+To eliminate ambiguity in variant interpretation when GT=0, all copy number haplotypes should be explicitly specified with $<$CNV$>$ alleles, even when copy number is 1.
```

Comment:
```suggestion
For example, a region on chr1 from position 101 to 130 (both inclusive) with allele-specific copy numbers of 1 and 2 can be represented as follows:

\footnotesize
\begin{verbatim}
	chr1 100 . T <CNV> . . END=130;SVLEN=30;CN=2 GT:CN 0/1:3
\end{verbatim}
\normalsize

To eliminate ambiguity in variant interpretation when GT=0, all copy number haplotypes should be explicitly specified with $<$CNV$>$ alleles, even when copy number is 1, as follows:

\footnotesize
\begin{verbatim}
	chr1 100 . T <CNV>,<CNV> . . END=130;SVLEN=30,30;CN=1,2 GT:CN 1/2:3
\end{verbatim}
\normalsize

All $<$CNV$>$ alleles in the same VCF record should have the same SVLEN.
```

There is a typo in the first sentence: the region on chr1 spans from 101 to 130 and not 300. I would also add one example using `GT=0` with the REF copy number meaning, and the disambiguation option with the two `<CNV>` ALT alleles.